lookup_plugins/manala_docker_applications.py

```diff
@@ -20,6 +20,7 @@ def run(self, terms, variables=None, **kwargs):
             'tty':         None,
             'command':     None,
             'volumes':     {},
+            'envs':        {},
```

Comment:
this composes one env, I would call it `env` instead. see e.g. 
https://docs.docker.com/compose/environment-variables/#setting-environment-variables-in-containers
https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Process/Process.php#L141